README.md

```diff
@@ -13,6 +13,23 @@ This module can be used to provision and configure [Context Based Restrictions](
 
 See in particular the [fscloud profile](./profiles/fscloud/) module that enables creating an opiniated account-level coarse-grained set of CBR rules and zones aligned with the "secure by default" principles.
 
+<!-- BEGIN OVERVIEW HOOK -->
+## Overview
+* [terraform-ibm-cbr](#terraform-ibm-cbr)
+* [Submodules](./modules)
```

Comment:
We probably should also move the profiles/fscloud module under the modules directory so that it is included in the overview (as well as surfaced in hashicorp)

Reply:
fscloud dir moved from profiles to modules dir